CONTRIBUTING.md

```diff
@@ -24,26 +24,29 @@ Current integrators (alphabetical order):
 Guidelines for pull requests
 ----------------------------
```

Comment:
Problem: when I started with Spoon contribution a year ago, I read these rules before I made my first PR to Spoon. But I understood it wrong - I though that PR must fulfill all these rules since the beginning, therefore I had a fear to make the first PR here ;-)

Solution: I suggest to add here something like: 

> This guideline is mandatory for PRs before they are merged to master. If you have anything interesting for Spoon community, then we welcome your PR even if it is not ideal since beginning. Spoon integrators will help You to fix remaining problems - if any. Your contribution is highly welcome! ;-)